### ISSUE TITLE
Unable see WCAG settings in configuration

### ISSUE THREAD

**Issue by Abubakar22** [Tanaguru/Tanaguru#179]
Hi Tanaguru team,

My team is looking for WCAG configuration in test and unable to find it, can you please let us know how to configure it.

It would be better if we get to know difference between RGAA and WCAG 

Below is the screenshot of my configuration 

![Tanaguru](https://user-images.githubusercontent.com/17588727/75920054-405ea700-5e24-11ea-91c5-2ed3cbe84281.png)

And Tanaguru website says it support 
https://www.tanaguru.com/en/services/supporting-web-accessibility/

**Comment by Abubakar22** [comment]
@rcharre can you please help me on this, Either my understanding is not right or I need more details about this specific configuration

**Comment by fhalna** [comment]
Hi @Abubakar22 ,
RGAA is based on the WCAG, currently only the RGAA standard is available. But, a dedicated WCAG 2.1 version will be available in September.

**Comment by Abubakar22** [comment]
Thank you @fhalna,

**RGAA is based on the WCAG** if possible can you please provide some supporting documentation for this

**Comment by fhalna** [comment]
you have this documentation http://rgaa.tanaguru.com/en/rgaa3-criteria.html

![image](https://user-images.githubusercontent.com/6985165/80398512-0657d180-88b8-11ea-80f4-6761c69dba75.png)

each criteria is mapping with WCAG 2.0
Next version of Tanaguru will be based on RGAA4 and WCAG 2.1
